desk/lib/bolt.hoon

```diff
@@ -39,9 +38,7 @@
   +*  this  .
       ag    ~(. yosh bowl)
       io    ~(. agentio bowl)
-      perms  ~(. permissions bowl state)
```

Comment:
`permissions` already has access to `state` since you pinned it to the subject *before* the `=<` further up.